src/pages/explore/masternodes/MasternodesBenefitsColumn.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  PassiveIncomeIcon,
+  PassiveIncomeIconHover,
+} from "@components/icons/assets/masternodesBenefits/PassiveIncomeIcon";
+import {
+  StakingRewardsIcon,
+  StakingRewardsIconHover,
+} from "@components/icons/assets/masternodesBenefits/StakingRewardsIcon";
+import {
+  NetworkSecurityIcon,
+  NetworkSecurityIconHover,
+} from "@components/icons/assets/masternodesBenefits/NetworkSecurityIcon";
+import {
+  GovernanceIcon,
+  GovernanceIconHover,
+} from "@components/icons/assets/masternodesBenefits/GovernanceIcon";
+import classNames from "classnames";
+import { useState } from "react";
+
+interface BlockchainFeatureItemProp {
+  title: string;
+  desc: string;
+}
+
+export function MasternodesBenefitsColumn({
+  items,
+  id,
+}: {
+  items: Array<BlockchainFeatureItemProp>;
+  id: string;
+}): JSX.Element {
+  return (
+    <div className="flex flex-1 flex-wrap gap-12 justify-between md:gap-y-[120px] lg:mr-[64px] lg:gap-y-16 lg:gap-x-10">
+      {items.map((item) => (
+        <FeatureIcon item={item} id={id} key={item.title} />
+      ))}
+    </div>
+  );
+}
+
+function FeatureIcon({
+  item,
+  id,
+}: {
+  item: BlockchainFeatureItemProp;
+  id: string;
+}) {
+  const Icon = iconMapping[item.title] as React.ElementType;
+  const IconOnHover = iconMapping[`${item.title}_hover`] as React.ElementType;
+  const [isHovering, setIsHovering] = useState(false);
+
+  const handleMouseOver = () => {
+    setIsHovering(true);
+  };
+
+  const handleMouseOut = () => {
+    setIsHovering(false);
+  };
+
+  return (
+    <div
+      className="masternode-benefits-feature-item flex flex-row md:flex-col w-full md:w-[306px] lg:w-[206px]"
+      onMouseOver={handleMouseOver}
+      onFocus={handleMouseOver}
+      onMouseOut={handleMouseOut}
+      onBlur={handleMouseOut}
+    >
+      <div className="w-[50px]">
+        {isHovering && Icon ? (
+          <IconOnHover className="w-[50px] md:w-fit h-[50px]" />
```

Comment:
let's not add hover icon as well, to have the same static behaviour as `/explore/dfi`